src/Plivo/Resources/Message/MessageInterface.php

```diff
@@ -211,6 +261,37 @@ public function __call($name_of_method, $arguments)
                 {
                     throw new PlivoResponseException($responseContents['error'], 0, null, $response->getContent() , $response->getStatusCode());
                 }
+        }
+        
+        elseif($name_of_method=='list')
+        {
+            if (empty($arguments[0]))
+            {
+                $args = [];
+            }
+            else
+            {
+                $args = $arguments[0];
```

Comment:
why one only arguments is considering on list?

Reply:
@narayana-plivo we are receiving all the parameters in our first array index.
```
$response = $client->messages->list([ 'limit' => 5,'offset' => 0]);
```

Reply:
is argument type should be array ?

Reply:
@narayana-plivo yeah, we are accepting argument as array, for both old & new messaging interface & List all messages.